task/operator-sdk-generate-bundle/0.1/operator-sdk-generate-bundle.yaml

```diff
@@ -12,6 +12,9 @@ spec:
     - name: channels
       description: Comma-separated list of channels the bundle belongs to
       default: alpha
+    - name: kustomize-dir
+      description:
```

Comment:
Do you want to add a description?

Reply:
Copied the CLI flag's description! Thanks Adam!!